src/components/RangeSlider/components/DualThumb/DualThumb.scss

```diff
@@ -15,14 +16,16 @@ $range-thumbs-border-active: rem(2px) solid color('indigo');
   position: relative;
   width: 100%;
   display: flex;
+  align-items: center;
```

Comment:
To fix prefix and suffix.

![screen shot 2019-02-08 at 7 10 48 pm](https://user-images.githubusercontent.com/6239480/52515813-5d9b5480-2bd5-11e9-822f-7d1d49f626db.jpg)